src/common.cc

```diff
@@ -96,6 +96,10 @@ namespace sharp {
       descriptor->rawWidth = AttrAsUint32(input, "rawWidth");
       descriptor->rawHeight = AttrAsUint32(input, "rawHeight");
     }
+    // Raw pixel input is premultiplied
+    if (HasAttr(input, "rawPremultiplied")) {
```

Comment:
If possible, it might be better to ensure `inputDescriptor.rawPremultiplied` is always set in the JavaScript code to avoid a conditional statement here.